beacon_node/eth2_libp2p/src/rpc/protocol.rs

```diff
@@ -569,6 +656,12 @@ impl<TSpec: EthSpec> std::fmt::Display for RPCRequest<TSpec> {
     }
 }
 
+impl<TSpec: EthSpec> std::fmt::Display for RpcRequestContainer<TSpec> {
```

Comment:
I dont think we need this either. It should just be an internal container. If we ever need to print it, we should print the whole thing and not pretend like its just an RpcRequest